keras/losses.py

```diff
@@ -4,12 +4,169 @@
 from __future__ import division
 from __future__ import print_function
 
+import abc
 import six
+
 from . import backend as K
+from .utils import losses_utils
 from .utils.generic_utils import deserialize_keras_object
 from .utils.generic_utils import serialize_keras_object
 
 
+class Loss(object):
+    """Loss base class.
+
+    To be implemented by subclasses:
+    * `call()`: Contains the logic for loss calculation using `y_true`, `y_pred`.
+
+    Example subclass implementation:
+    ```
+    class MeanSquaredError(Loss):
+    def call(self, y_true, y_pred):
+      y_pred = ops.convert_to_tensor(y_pred)
+      y_true = math_ops.cast(y_true, y_pred.dtype)
+      return K.mean(math_ops.square(y_pred - y_true), axis=-1)
+    ```
+
+    Args:
```

Comment:
Use convention `# Arguments` --

- start with `#`
- no `:`

This applies to all new docstrings.